crates/experimentation_platform/tests/experimentation_tests.rs

```diff
@@ -11,8 +11,8 @@ enum Dimensions {
     CLIENT(String),
 }
 
-fn single_dimension_ctx_gen(value: Dimensions) -> serde_json::Value {
-    match value {
+fn single_dimension_ctx_gen(value: Dimensions) -> Option<serde_json::Value> {
```

Comment:
why return optional, if you are explicitly adding Some at the return statement.

Reply:
Hey because the type required is Option<Value>